src/main/java/com/iota/iri/service/API.java

```diff
@@ -760,10 +997,20 @@ private AbstractResponse getInclusionStatesStatement(final List<String> transact
                 }
             }
         }
+        
+        // Finds the lowest tips index, or 0
         int minTipsIndex = tipsIndex.stream().reduce((a,b) -> a < b ? a : b).orElse(0);
+        
+        // If minTipsIndex is 0 or lower, we can't check transactions against snapshots because there were no tips
```

Comment:
```suggestion
        // If the lowest tips index (minTipsIndex) is 0 (or lower), we can't check transactions against snapshots because there were no tips
```
I don't understand the last part.
it means that none of the tips was ever confirmed.

Reply:
oh reading the 2nd line I get it :)
worth swapping order.